config.py

```diff
@@ -38,6 +40,15 @@ class Config:
     DECRYPTION_KEY_PASSPHRASE = os.getenv('DECRYPTION_KEY_PATH', 'test')
 
     ACTION_RULE_DELAY_MINUTES = float(os.getenv('ACTION_RULE_DELAY_MINUTES', 0.3))
-    PRINT_FILE_TIME_LIMIT_MINUTES = float(os.getenv('PRINT_FILE_TIME_LIMIT_MINUTES', 10))
+    PRINT_FILE_TIME_LIMIT_MINUTES = float(os.getenv('PRINT_FILE_TIME_LIMIT_MINUTES', 120))
     SAMPLE_FILE_PATH = Path(os.getenv('SAMPLE_FILE_PATH',
-                                      PROJECT_PATH.joinpath('resources', 'sample_files', '1_per_treatment_code.csv')))
+                                      PROJECT_PATH.joinpath('resources', 'sample_files',
+                                                            '100_per_treatment_code.csv')))
+
+    # SAMPLE_FILE_PATH = Path(os.getenv('SAMPLE_FILE_PATH',
+    #                                   PROJECT_PATH.joinpath('resources', 'sample_files',
+    #                                                         '1000_per_treatment_code.csv')))
+
+    # SAMPLE_FILE_PATH = Path(os.getenv('SAMPLE_FILE_PATH',
+    #                                   PROJECT_PATH.joinpath('resources', 'sample_files',
+    #                                                         '350000_sample_file.csv')))
```

Comment:
Commented out code

Reply:
I've left those on purpose at the moment to easily switch between files whilst the tests are developed.   Not sure what we're want the default one to be.

Reply:
Should we not be doing that with the environment variable, rather than with commented out code?

Reply:
don't know really, got rid of them now